ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -329,10 +327,21 @@ describe('a4a_config hash param parsing', () => {
       },
       navigator: window.navigator,
     };
+    win.document.defaultView = win;
+    const ampdocService = installDocService(win, /* isSingleDoc */ true);
+    ampdoc = ampdocService.getAmpDoc();
     events = {};
     installPlatformService(win);
     documentStateFor(win);
+    const attrs = {};
+    element = {
```

Comment:
Yes. This is unfortunate. I'm independently working on test rules to avoid this discrepancy. But it's out of scope for this PR.